### PR TITLE
Refactor account for Facebook OAuth

### PR DESCRIPTION
**Pull request by dawntomm**
## New Behavior
Refactor account module for Facebook OAuth.

Previously, we use a 3rd party package for Facebook Graph API. However, importing the whole package just for retrieving user account information seems unnecessary. This pull request provides a better way of sending http GET request to facebook API to retrieve user information without using 3rd party package.

### CODE REVIEW

File: backend/app/adapter/facebook/account.go [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
 
 import (
+	"net/http"
+	"net/url"
+
+	"github.com/short-d/app/fw"
 	"github.com/short-d/short/app/entity"
 	"github.com/short-d/short/app/usecase/service"
-
-	fb "github.com/huandu/facebook"
```
Comment (alldroll) [inline backend/app/adapter/facebook/account.go]: Please run `go mod tidy` because we don't need this dependency anymore

Comment (dawntomm) [reply backend/app/adapter/facebook/account.go]: addressed comment

**Comment by alldroll** [comment]
@byliuyang Please take a look at this PR for double-checking. Thanks:)

**Comment by byliuyang** [comment]
Hi @dawntomm , thank you very much for working on this cool refactor! Can we add integration test for this PR?

**Comment by dawntomm** [comment]
> Hi @dawntomm , thank you very much for working on this cool refactor! The code looks good! Can we add integration test for this PR?

just added integration!

File: backend/app/adapter/facebook/account_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
+package facebook
```
Comment (byliuyang) [inline backend/app/adapter/facebook/account_integration_test.go]: Need build tag here

**Pull request merged**